Resources/doc/cookbook/breadcrumbs.md

```diff
@@ -0,0 +1,162 @@
+# Breadcrumbs
+
+[Go back to Table of contents][back-to-index]
+
+-----
+
+It is possible to add breadcrumbs to your admin generator when following this cookbook article.
+
+![Breadcrumbs preview](../img/cookbook-breadcrumbs.png)
+
+### 1. Install suggested breadcrumb bundle
+Use composer to install `"cnerta/breadcrumb-bundle": "^2.0"` and add the following to your configuration:
+
+```yaml
+cnerta_breadcrumb:
+    twig:
+        template:	breadcrumbs.html.twig
+```
+
+### 2. Create your personal breadcrumbs template
+
+You can customize the example below if needed. The file should be located in `app/Resources/views/breadcrumbs.html.twig`:
+```twig
+{% block root %}
+  {% if items|length > 1 %}
+    {% spaceless %}
+      <ol id="breadcrumbs" class="breadcrumb hidden-xs hidden-print" itemscope itemtype="http://data-vocabulary.org/Breadcrumb">
+  	  {% for item in items %}
+          {% if loop.last %}{%set itemClass = "active"%}{% endif %}
+          <li{% if itemClass is defined and itemClass|length %} class="{{ itemClass }}"{% endif %}{% if not(loop.first) %} itemprop="child"{% endif %}>
+            {% if item.uri and not loop.last %}<a href="{{ item.uri }}" itemprop="url"{% if linkRel is defined and linkRel|length %} rel="{{ linkRel }}"{% endif %}>{% endif %}
+            {% set icon = item.item.getExtra('icon') %}
+            {% if icon is defined and icon is not empty and not loop.last %}<i class="{{ icon }}"></i>{% endif %}
+            <span itemprop="title">{{ item.label }}</span>
+            {% if item.uri and not loop.last %}</a>{% endif %}
+  		</li>
+  	  {% endfor %}
+      </ol>
+    {% endspaceless %}
+  {% endif %}
+{% endblock root %}
+```
+
+### 4. Update the admingenerator layout
+```twig
+{% block page_header %}
+  <section class="content-header">
+    {% block page_header_content '' %}
+    {% block page_header_breadcrumbs %}{{ cnerta_breadcrumb_render('AppBundle:Builder:breadcrumbMenu') }}{% endblock page_header_breadcrumbs %}
+  </section>
+{% endblock page_header %}
+```
+
+### 5. Building the breadcrumbbs tree
+
+Add the following code to your menu builder to build the breadcrumbs tree (`src/AppBundle/Menu/Builder.php`):
+```php
+public function breadcrumbMenu(FactoryInterface $factory, array $options)
+{
+    $primaryKey = $this->container->get('request')->get('pk', $this->container->get('request')->get('id', 1));
+    $menu = $factory
+        ->createItem('main', array('label' => 'Главная', 'route' => 'homepage'))
+        ->setExtra('icon', 'fa fa-dashboard');
+    $login = $menu->addChild('Идентификация', array('route' => 'fos_user_security_login'))
```

Comment:
@bobvandevijver @loostro Not at all! I could translate it to common language...  ))